frontend/src/app/offers/offers-box/offers-box.component.html

```diff
@@ -0,0 +1,11 @@
+<div *ngIf="(offers$ | async)?.length==0; else offersExist">
```

Comment:
> This approach has a few flaws, the first and most obvious is being potentially exposed to multiple, unwanted, subscriptions (previously mentioned up top) that initiate requests.

See how to handle that here: https://toddmotto.com/angular-ngif-async-pipe

Reply:
You can even change order here and render list from `ngIf` and in `else` block show info, that no offers were yet joined.